soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -7923,6 +7923,13 @@ s32 func_8084269C(PlayState* play, Player* this) {
     return 0;
 }
 
+// todo: put this somewhere better idk where yet
+typedef enum {
+    DEKU_STICK_STANDARD,
+    DEKU_STICK_UNBREAKABLE,
+    DEKU_STICK_UNBREAKABLE_AND_ALWAYS_ON_FIRE,
+} DekuStickCheat;
```

Comment:
idk where to put this, anyone have a good suggestion?

Reply:
mods.h? Not my favorite but better than 7000 lines into player.c at least

Reply:
I agree with putting it in `mods.h`. It seems like the best place to put it.

Reply:
i moved it to `enhancementTypes.h`

https://github.com/HarbourMasters/Shipwright/blob/c0e1728af5a17416dc2ba896de7837cc0c4c7586/soh/soh/Enhancements/enhancementTypes.h#L72-L76